persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_96_redis/persist/model.bal

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 202e WSO2 LLC. (http://www.wso2.org).
```

Comment:
```suggestion
// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org).
```

Reply:
This has been fixed in https://github.com/ballerina-platform/persist-tools/pull/333/commits/a47e9c401642d1ee32077e4b83f1420855854ec4